Minor update (#179)
floydhub/floyd-docs: docs/guides/serving.md

You can also view a complete example [here](https://github.com/floydhub/fast-style-transfer/blob/master/app.py).

!!! important "Pay attention to Race Conditions!"

>     The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of race condition: remember to lock the resources every time you will need to write and read from them. 

Reviewer: memory read write is ok, because each worker run in its own process. It's fairly safe to restrict it to disk resource for now.

Author: Ok, I've tried to be as generic as possible. I'll restrict it to disk resources then.

>     The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of race condition: remember to lock the disk resources every time you will need to write and read from them. 

Reviewer 2: Can you provide an example of how to lock the disk resources?

Reviewer: we should just avoid doing it, it's not a good idea to write to disk for serving anyway, it will slow down the start up time and I can't think of a good use-case for this.

Author: So do you think that it's better to not mention this at all? Maybe we can reformulate the sentence  by saying that it is considered as an "anti-pattern" (_probably is not right_) to write on disk while serving, but if they need it, they need to pay attention to race conditions. Sounds good?

Reviewer: Yeah, this sounds good 👍

>     The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of [race conditions](https://en.wikipedia.org/wiki/Race_condition#software)! Writing on disk when serving model is considered an anti-pattern, but if your use-case require it, please remember to lock the disk resources every time you will need to write and read from them. 

Reviewer 3: ```suggestion
    The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of [race conditions](https://en.wikipedia.org/wiki/Race_condition#software)! Writing on disk when serving a model is considered not recommended, but if your use-case requires it, remember to lock the disk resources every time you write to them. 
```
